docs/install_dependencies.md

```diff
@@ -10,6 +10,7 @@
   + xz-utils
   + lscpu
   + cargo, rustc (version \~ latest)
+  + libclang (version >= 9), for bindgen
```

Comment:
nit: Adding "for bindgen" would make me think that this is a requirement for some optional feature, but that may just be me.